src/components/LandingPage/LandingPage.js

```diff
@@ -1,22 +1,45 @@
 import ResultCard from "../ResultCard/ResultCard";
 import "./LandingPage.css"
 
-const LandingPage = () => {
+const LandingPage = ({ user, addFavorite, deleteFavorite, featured }) => {
+  const featBusinesses = featured.map(business => {
+    const { id } = business;
```

Comment:
Seems there is a "semi-rare" page refresh bug that crashes the app with the error seen below. Perhaps there is something with how the randomized businesses are generated in App? Looking into it.

![Screenshot 2022-12-10 at 9 04 41 AM](https://user-images.githubusercontent.com/100455148/206864351-0454e7b4-c791-4525-a98c-f4ab8a621d4c.png)

Reply:
Good catch! Solved it - accidentally put business.data.length instead of business.data.length - 1 as the max boundary -- I'll push now!

Reply:
Awesome - re-reviewing and merging!